docs/arch/adr-008-weave-to-flannel-migration.md

```diff
@@ -0,0 +1,39 @@
+# ADR 8: Weave to Flannel Migration
+
+## Context
+
+As proposed in ADR 6, the future CNI of kURL will be Flannel.
+However, almost all existing installations are running Weave.
+
+A migration path must be provided for those instances.
+
+There were two general proposals for a migration path.
+One was to use Multus-CNI to allow using both Flannel and Weave at the same time before stripping out Weave, allowing an upgrade while only taking down one node at once.
+The other was to run a migration that removed Weave from all instances simultaneously before installing Flannel.
+This would be far simpler, not require an additional IP range, and not require Multus to remain in the cluster going forwards.
+It would also require 'stopping the world' and cause downtime.
```

Comment:
What about we have here:

## Proposal

You describe what do you think is the best option.

Then, we add another section

## Alternatives

Also, I think we could have on this one `Risks and Mitigations` where we describe the downtime and what are the mitigations ( script that does the work in parallel to reduce the time and communication )